special-pages/pages/new-tab/app/favorites/components/Favorites.js

```diff
@@ -242,12 +243,17 @@ function Inner({ rows, safeAreaRef, rowHeight, add }) {
             { signal: controller.signal },
         );
 
-        // when the content-tube grows, re-calc the layout
-        const resizer = new ResizeObserver(() => {
-            requestAnimationFrame(() => {
-                updateGlobals(mainScroller.scrollTop);
-                setVisibleRowsForOffset(mainScroller.scrollTop);
-            });
+        let lastHeight;
+        const resizer = new ResizeObserver((entries) => {
+            const first = entries[0];
+            if (!first) return;
+            if (first.contentRect.height !== lastHeight) {
+                lastHeight = first.contentRect.height;
+                requestAnimationFrame(() => {
+                    updateGlobals(mainScroller.scrollTop);
+                    setVisibleRowsForOffset(mainScroller.scrollTop);
+                });
+            }
```

Comment:
This change prevents the favorites grid re-calcualating when the drawer animates